### ISSUE TITLE
 error in halui docs 

### ISSUE THREAD

**Issue by hansu** [LinuxCNC/linuxcnc#2850]
Reported here: https://forum.linuxcnc.org/32-documents/51387-error-in-halui-docs#290960

I think it would be good to change the pin name in the sources to follow LinuxCNC's convention to use hyphens for that.
To not break the configs we can use the update-ini script to update the config.
What do you think @andypugh ?

**Comment by andypugh** [comment]
I agree that we should change the actual pin names and update the update-ini script. 
There should also be a mention in the "updating linuxcnc" section of the docs that mentions this. 
I further think that it should probably be a 2.10 change rather than go into 2.9.

**Comment by hansu** [comment]
> I further think that it should probably be a 2.10 change rather than go into 2.9.

I also thought in this direction to do this change at least in the next 2.9 release, so 2.9.3. But 2.10 would be also fine.

**Comment by andypugh** [comment]
Future 2.9 releases should be bugfixes only (arguably this is a bug fix, but...)
My main reason for preferring to push it to 2.10 is that I don't think that users will be expecting config changes (or to see the update_in script run) after a minor update.
